apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -82,13 +82,23 @@ export default class ResearchOutputs implements ResearchOutputController {
       )
       .join(' or ');
 
-    const filterQ = (filter || [])
-      .reduce(
-        (acc: string[], word: string) =>
-          acc.concat([`data/type/iv eq '${word}'`]),
-        [],
-      )
-      .join(' or ');
+    let filterQ: string;
```

Comment:
the code below could be function :)

Reply:
yeah i know it could but im so annoyed when i see all these utility functions in this repo just so the code is more 'functional' that i almost did this on purpose. I guess i will refactor though

Reply:
I think it helps break it down into single purpose functions